frontend/packages/knative-plugin/src/components/add/event-sources/EventSourceSection.tsx

```diff
@@ -48,6 +69,11 @@ const EventSourceSection: React.FC<EventSourceSectionProps> = ({
       />
     </>
   );
+  const formSchema: JSONSchema6 = {
```

Comment:
will it make sense to make use of `useCallback` here or have inside switch block for `EventSources.KameletBinding`